tests/unit/JsonUtilTests.cpp

```diff
@@ -88,3 +88,23 @@ TEST(JsonUtils, integralValueAs)
     auto const stringJson = boost::json::value("not a number");
     EXPECT_THROW(util::integralValueAs<int>(stringJson), std::logic_error);
 }
+
+TEST(JsonUtils, getLedgerIndex)
+{
+    auto const emptyJson = boost::json::value();
+    EXPECT_THROW(util::getLedgerIndex(emptyJson), std::logic_error);
```

Comment:
Maybe it would be better to not throw but return `nullopt` in this case? @godexsoft, @mathbunnyru wdyt?

Reply:
Maybe an `expected` then?